### PR TITLE
Add new brands

### LINKED ISSUE #2351: Add entry for: Bembos
First issue. A suggestion to add Peruvian franchises. Thanks in advance.
[Bembos Website](https://www.bembos.com.pe/)
[Wikipedia](https://en.wikipedia.org/wiki/Bembos)
`brand=Bembos`
`amenity=fast_food`
`cuisine=burger`

### LINKED ISSUE #2352: Add entry for: Banco de la Nación (Perú)
Do not confuse with the bank from Argentina.
[Banco de la Nación Website](https://www.bn.com.pe/)
[Wikipedia](https://es.wikipedia.org/wiki/Banco_de_la_Naci%C3%B3n_(Per%C3%BA))
`brand=Banco de la Nación`
`amenity=bank`

### LINKED ISSUE #2353: Add entry for: Serpost
[Serpost Website](https://www.serpost.com.pe)
[Wikipedia](https://es.wikipedia.org/wiki/Servicios_Postales_del_Per%C3%BA)
`brand=Serpost`
`amenity=post_office`

### LINKED ISSUE #2354: Add entry for: China Wok
[China Wok Website](https://www.chinawok.com.pe)
[Wikipedia](https://es.wikipedia.org/wiki/China_Wok)
`brand=China Wok`
`amenity=fast_food`
`cuisine=peruvian`

### LINKED ISSUE #2355: Add entry for: Marathon Sports
[Marathon Sports Website](https://www.marathon.store/)
[Wikipedia](https://es.wikipedia.org/wiki/Marathon_Sports)
`brand=Marathon Sports`
`shop=sports`

### LINKED ISSUE #2356: Add entry for: Oechsle
[Oechsle Website](https://www.oechsle.pe/)
[Wikipedia](https://es.wikipedia.org/wiki/Oechsle)
`brand=Oechsle`
`shop=department_store`

### LINKED ISSUE #2357: Add entry for: Sodimac
[Sodimac Website](http://www.sodimac.com/)
[Wikipedia](https://es.wikipedia.org/wiki/Sodimac)
`brand=Sodimac`
`shop=doityourself`

### LINKED ISSUE #2358: Add entry for: D'Onofrio
[D'Onofrio Website](https://www.nestle.com.pe/nuestros-productos/helados)
[Wikipedia](https://es.wikipedia.org/wiki/D%27Onofrio)
`brand=D'Onofrio`
`amenity=ice_cream`

Also written as "Donofrio".

### PR DESCRIPTION
**Pull request by sguinetti**
See issues  (Peruvian brands).
Closes #2358, closes #2357, closes #2356, closes #2355, closes #2354, closes #2353, closes #2352 and closes #2351

Also:
* New: GMO optician shop (Latina America)
* New: "Pardos Chicken" (Peru)
* Update: "Lawson Store 100" (with english name)
* Update: "Puma" (add shop="sport" in match)
* New: "Natural Lawson" (with Japan name)
* Change "Banco de la Nación" - > "Banco de la Nación (Argentina)" with CountryCode to "ar"
* Cencosud and Intercorp brands.
* Some strings in filters.json. Like "Cebiche", "Antichuchos".

If there are problems in this file, let me know to correct them.

### CODE REVIEW

**Comment by tas50** [comment]
Puma should be Shop clothing to match other sports clothing shops such as Nike

**Comment by sguinetti** [comment]
Sure. The detail is that I added to the code match and brand:
```
"shop/clothes|Puma": {
		"count": 67,
		"match": ["shop/sports|Puma"],
		"nomatch": ["amenity/fuel|Puma"],
		"tags": {
			"brand": "Puma",
			"brand:wikidata": "Q7203029",
			"brand:wikipedia": "en:Puma (brand)",
			"name": "Puma",
			"shop": "clothes"
		}
},
```

**Comment by tas50** [comment]
@sguinetti Something happened with the PR here and it's now changing a huge number of lines that introduces a conflict.

**Comment by sguinetti** [comment]
Yes. Maybe when editing from another program it seems that the problems start. I will try to do several pull requests to see how to fix it.

**Comment by sguinetti** [comment]
@tas50 Finish. In addition, the request contains the patches #2361  #2359

**Comment by Adamant36** [comment]
@sguinetti, if put "closes" before the issue numbers it will automatically close the issues they are associated with when the PR is merged. It saves a lot of time because  we don't have to go through and search for them to manually close them. 

https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by Adamant36** [comment]
Btw, is brand:en": "NATURAL LAWSON" suppose to be in all capitals? And one more thing, if you do npm run build in the terminal before you do the commit it will rebuild it and check for any errors there might be in your code. Along with getting with rid of the trailing space at the end. Its not necessary, but good to do if your adding a bunch of stuff just in case you made a mistake somewhere that your not aware of. I've had that happen a few times myself. 

Other then that, thanks for the PR and also to @tas50 for reviewing it :)

**Comment by sguinetti** [comment]
@Adamant36 In the japanese Wikipedia is branded in capital letters For example [Lawson](https://ja.wikipedia.org/wiki/%E3%83%AD%E3%83%BC%E3%82%BD%E3%83%B3). I'm checking with jsonlint.com online editor instead of npm. Sorry for the link but I had to explain how I check the JSON code.

**Comment by Adamant36** [comment]
> I'm checking with jsonlint.com online editor instead of npm

Your still doing it in a terminal with npm setup though right? Otherwise how do you do git and all that? I'm completely ignorant to cloud services and how they work for this kind of thing.

**Comment by sguinetti** [comment]
The important thing is that the text says "All checks have passed" or something similar in the pull request. Right?

**Comment by Adamant36** [comment]
I'm pretty sure npm run build can fail but then Travis will still pass because its a different check system. Plus, npm run build actually fixings certain things on its own, like the trailing space. Whereas Travis doesn't (it will just flag it).

**Comment by Adamant36** [comment]
Btw, you shouldn't go through and close a bunch of issues for PR's that aren't merged yet. Otherwise, you risk closing them when there might be a reason the PR doesn't get merged. Its much better just to "closes" in your original PR message so it does it itself.

**Comment by sguinetti** [comment]
@Adamant36 Thanks for the tip. I understood as "close all the issues". I will put these keyworks in the first message.

**Comment by tas50** [comment]
@sguinetti You really want to run npm run build every time before you commit. It not only detects and corrects poorly formatted and incorrect JSON, but it also detects incorrectly formatted data. You can skip the whole jsonlint.com route by just running that locally.

File: config/canonical.json [hunk 1]
```diff
@@ -10964,7 +11018,7 @@
   },
   "amenity/fuel|Puma": {
     "count": 604,
-    "nomatch": ["shop/clothes|Puma"],
+    "nomatch": ["shop/clothes|Puma","shop/sports|Puma"],
```
Comment (tas50) [inline config/canonical.json]: You can remove the nomatch for sports|Puma now

File: config/canonical.json [hunk 2]
```diff
@@ -35957,4 +36154,4 @@
       "tourism": "motel"
     }
   }
-}
\ No newline at end of file
+}
```
Comment (tas50) [inline config/canonical.json]: If you run npm run build you don't end wup with the new line change to the PR. It removes the blank line that most text editors try to add back.

File: config/canonical.json (hunk 1 above)
Comment (sguinetti) [reply config/canonical.json]: Fixed

**Comment by sguinetti** [comment]
I installed npm and used with terminal. The result is

```
Warning - Entries in `canonical.json` matched to other entries in `canonical.json`:
To resolve these, remove the worse entry and add "match" property on the better entry.
  "amenity/fast_food|Dairy Queen" -> matches? -> "amenity/fast_food|DQ Grill & Chill"
total 1

Warning - Missing tags for entries in `canonical.json`:
To resolve these, add the missing tag.
  "amenity/fast_food|Ali Baba" -> missing tag? -> "cuisine"
  "amenity/fast_food|Pinulito" -> missing tag? -> "cuisine"
  "amenity/fast_food|Pollo Granjero" -> missing tag? -> "cuisine"
  "amenity/fast_food|Γρηγόρης" -> missing tag? -> "cuisine"
  "amenity/fast_food|Крошка Картошка" -> missing tag? -> "cuisine"
  "amenity/fast_food|Теремок" -> missing tag? -> "cuisine"
  "amenity/fast_food|すき家" -> missing tag? -> "cuisine"
  "amenity/fast_food|なか卯" -> missing tag? -> "cuisine"
  "amenity/fast_food|はま寿司" -> missing tag? -> "cuisine"
  "amenity/fast_food|ほっかほっか亭" -> missing tag? -> "cuisine"
  "amenity/fast_food|ほっともっと" -> missing tag? -> "cuisine"
  "amenity/fast_food|オリジン弁当" -> missing tag? -> "cuisine"
  "amenity/fast_food|吉野家" -> missing tag? -> "cuisine"
  "amenity/fast_food|富士そば" -> missing tag? -> "cuisine"
  "amenity/fast_food|幸楽苑" -> missing tag? -> "cuisine"
  "amenity/fast_food|松屋" -> missing tag? -> "cuisine"
  "amenity/fast_food|肯德基" -> missing tag? -> "cuisine"
  "amenity/restaurant|100 Montaditos" -> missing tag? -> "cuisine"
  "amenity/restaurant|Autogrill" -> missing tag? -> "cuisine"
  "amenity/restaurant|Bella Italia" -> missing tag? -> "cuisine"
  "amenity/restaurant|Campanile" -> missing tag? -> "cuisine"
  "amenity/restaurant|Chifa" -> missing tag? -> "cuisine"
  "amenity/restaurant|Comedor" -> missing tag? -> "cuisine"
  "amenity/restaurant|Comida China" -> missing tag? -> "cuisine"
  "amenity/restaurant|Dolce Vita" -> missing tag? -> "cuisine"
  "amenity/restaurant|Gusto" -> missing tag? -> "cuisine"
  "amenity/restaurant|Le Bistrot" -> missing tag? -> "cuisine"
  "amenity/restaurant|Little Italy" -> missing tag? -> "cuisine"
  "amenity/restaurant|Loving Hut" -> missing tag? -> "cuisine"
  "amenity/restaurant|Parrilla" -> missing tag? -> "cuisine"
  "amenity/restaurant|Sizzler" -> missing tag? -> "cuisine"
  "amenity/restaurant|Sunset Grill" -> missing tag? -> "cuisine"
  "amenity/restaurant|Toby Carvery" -> missing tag? -> "cuisine"
  "amenity/restaurant|Vips" -> missing tag? -> "cuisine"
  "amenity/restaurant|Евразия" -> missing tag? -> "cuisine"
  "amenity/restaurant|Планета Суши" -> missing tag? -> "cuisine"
  "amenity/restaurant|Тануки" -> missing tag? -> "cuisine"
  "amenity/restaurant|Якитория" -> missing tag? -> "cuisine"
  "amenity/restaurant|はなまるうどん" -> missing tag? -> "cuisine"
  "amenity/restaurant|びっくりドンキー" -> missing tag? -> "cuisine"
  "amenity/restaurant|やよい軒" -> missing tag? -> "cuisine"
  "amenity/restaurant|ガスト" -> missing tag? -> "cuisine"
  "amenity/restaurant|ココス" -> missing tag? -> "cuisine"
  "amenity/restaurant|サイゼリヤ" -> missing tag? -> "cuisine"
  "amenity/restaurant|ジョイフル" -> missing tag? -> "cuisine"
  "amenity/restaurant|ジョナサン" -> missing tag? -> "cuisine"
  "amenity/restaurant|ジョリーパスタ" -> missing tag? -> "cuisine"
  "amenity/restaurant|デニーズ" -> missing tag? -> "cuisine"
  "amenity/restaurant|バーミヤン" -> missing tag? -> "cuisine"
  "amenity/restaurant|ビッグボーイ" -> missing tag? -> "cuisine"
  "amenity/restaurant|リンガーハット" -> missing tag? -> "cuisine"
  "amenity/restaurant|ロイヤルホスト" -> missing tag? -> "cuisine"
  "amenity/restaurant|八方雲集" -> missing tag? -> "cuisine"
  "amenity/restaurant|和食さと" -> missing tag? -> "cuisine"
  "amenity/restaurant|夢庵" -> missing tag? -> "cuisine"
  "amenity/restaurant|大阪王将" -> missing tag? -> "cuisine"
  "amenity/restaurant|天下一品" -> missing tag? -> "cuisine"
  "amenity/restaurant|永和豆漿" -> missing tag? -> "cuisine"
  "amenity/restaurant|餃子の王将" -> missing tag? -> "cuisine"
  "amenity/vending_machine|Automat ŚKUP" -> missing tag? -> "vending"
  "amenity/vending_machine|BKK-automata" -> missing tag? -> "vending"
  "amenity/vending_machine|Dog-Station" -> missing tag? -> "vending"
  "amenity/vending_machine|Milchtankstelle" -> missing tag? -> "vending"
  "amenity/vending_machine|ParkPlus" -> missing tag? -> "vending"
  "amenity/vending_machine|Redbox" -> missing tag? -> "vending"
  "amenity/vending_machine|Robidog" -> missing tag? -> "vending"
  "amenity/vending_machine|Tobaccoland" -> missing tag? -> "vending"
  "amenity/vending_machine|VVO Fahrausweise" -> missing tag? -> "vending"
total 68

Warning - Potential duplicate names in `canonical.json`:
To resolve these, remove the worse entry and add "match" property on the better entry.
To suppress this warning for entries that really are different, add a "nomatch" property on both entries.
  "amenity/fuel|Sinopec Fuel" -> duplicates? -> "amenity/fuel|Sinopec"
  "amenity/payment_centre|Abitab" -> duplicates? -> "amenity/bureau_de_change|Abitab"
  "amenity/payment_terminal|ПриватБанк" -> duplicates? -> "amenity/bank|ПриватБанк"
  "amenity/post_office|PTT" -> duplicates? -> "amenity/fuel|PTT"
  "amenity/restaurant|Dolce Vita" -> duplicates? -> "amenity/cafe|Dolce Vita"
  "shop/beverages|Бристоль" -> duplicates? -> "shop/alcohol|Бристоль"
  "shop/chemist|Yves Rocher" -> duplicates? -> "shop/beauty|Yves Rocher"
  "shop/clothes|西松屋" -> duplicates? -> "shop/baby_goods|西松屋"
  "shop/confectionery|Jeff de Bruges" -> duplicates? -> "shop/chocolate|Jeff de Bruges"
  "shop/confectionery|Leonidas" -> duplicates? -> "shop/chocolate|Leonidas"
  "shop/convenience|Alimentation générale" -> duplicates? -> "shop/convenience|Alimentation Générale"
  "shop/convenience|Almacén" -> duplicates? -> "shop/convenience|Almacen"
  "shop/convenience|Best-One" -> duplicates? -> "shop/convenience|Best One"
  "shop/convenience|Casino Shop" -> duplicates? -> "shop/convenience|Casino"
  "shop/convenience|Kangaroo Express" -> duplicates? -> "shop/convenience|Kangaroo"
  "shop/convenience|Pulpería" -> duplicates? -> "shop/convenience|Pulperia"
  "shop/convenience|RaceTrac" -> duplicates? -> "amenity/fuel|Race Trac"
  "shop/convenience|Rite Aid" -> duplicates? -> "shop/chemist|Rite Aid"
  "shop/convenience|Supérette" -> duplicates? -> "shop/convenience|Superette"
  "shop/convenience|Tesco Express" -> duplicates? -> "shop/convenience|Tesco"
  "shop/convenience|abc" -> duplicates? -> "shop/convenience|ABC"
  "shop/convenience|best-one" -> duplicates? -> "shop/convenience|Best-One"
  "shop/convenience|Épicerie" -> duplicates? -> "shop/convenience|Epicerie"
  "shop/convenience|Берёзка" -> duplicates? -> "amenity/cafe|Берёзка"
  "shop/convenience|Бристоль" -> duplicates? -> "shop/beverages|Бристоль"
  "shop/convenience|Гурман" -> duplicates? -> "amenity/cafe|Гурман"
  "shop/convenience|Каравай" -> duplicates? -> "shop/bakery|Каравай"
  "shop/convenience|Кристалл" -> duplicates? -> "shop/alcohol|Кристалл"
  "shop/convenience|Кулинария" -> duplicates? -> "amenity/cafe|Кулинария"
  "shop/convenience|Лакомка" -> duplicates? -> "amenity/cafe|Лакомка"
  "shop/convenience|Минимаркет" -> duplicates? -> "shop/convenience|Мини-маркет"
  "shop/convenience|Оазис" -> duplicates? -> "amenity/cafe|Оазис"
  "shop/convenience|Радуга" -> duplicates? -> "amenity/pharmacy|Радуга"
  "shop/convenience|Родны кут" -> duplicates? -> "shop/convenience|Родны Кут"
  "shop/convenience|Сказка" -> duplicates? -> "amenity/cafe|Сказка"
  "shop/convenience|Смак" -> duplicates? -> "amenity/cafe|Смак"
  "shop/convenience|Суши Wok" -> duplicates? -> "amenity/fast_food|Суши Wok"
  "shop/convenience|Фортуна" -> duplicates? -> "amenity/cafe|Фортуна"
  "shop/convenience|Хуторок" -> duplicates? -> "amenity/cafe|Хуторок"
  "shop/copyshop|Epicier" -> duplicates? -> "shop/convenience|Epicier"
  "shop/cosmetics|Marionnaud" -> duplicates? -> "shop/beauty|Marionnaud"
  "shop/cosmetics|Yves Rocher" -> duplicates? -> "shop/chemist|Yves Rocher"
  "shop/cosmetics|Мила" -> duplicates? -> "shop/chemist|Мила"
  "shop/cosmetics|Остров чистоты" -> duplicates? -> "shop/chemist|Остров чистоты"
  "shop/dairy|Braum's" -> duplicates? -> "amenity/fast_food|Braum's"
  "shop/deli|ほっともっと" -> duplicates? -> "amenity/fast_food|ほっともっと"
  "shop/department_store|Argos" -> duplicates? -> "shop/catalogue|Argos"
  "shop/department_store|Marks & Spencer" -> duplicates? -> "shop/clothes|Marks & Spencer"
  "shop/department_store|Épicerie" -> duplicates? -> "shop/convenience|Épicerie"
  "shop/electronics|BCC" -> duplicates? -> "amenity/bank|Bcc"
  "shop/electronics|DNS" -> duplicates? -> "shop/computer|DNS"
  "shop/furniture|Aaron's" -> duplicates? -> "shop/electronics|Aaron's"
  "shop/hairdresser|Виктория" -> duplicates? -> "amenity/cafe|Виктория"
  "shop/hairdresser|Елена" -> duplicates? -> "shop/convenience|Елена"
  "shop/hardware|Würth" -> duplicates? -> "shop/doityourself|Würth"
  "shop/hardware|Мастер" -> duplicates? -> "shop/doityourself|Мастер"
  "shop/hardware|Сантехника" -> duplicates? -> "shop/bathroom_furnishing|Сантехника"
  "shop/interior_decoration|Casa" -> duplicates? -> "shop/furniture|Casa"
  "shop/kiosk|Pulpería" -> duplicates? -> "shop/convenience|Pulpería"
  "shop/newsagent|Edicola" -> duplicates? -> "shop/kiosk|Edicola"
  "shop/newsagent|Maison de la Presse" -> duplicates? -> "shop/books|Maison de la Presse"
  "shop/newsagent|Relay" -> duplicates? -> "shop/kiosk|Relay"
  "shop/newsagent|WHSmith" -> duplicates? -> "shop/books|WHSmith"
  "shop/newsagent|Белсоюзпечать" -> duplicates? -> "shop/kiosk|Белсоюзпечать"
  "shop/newsagent|Роспечать" -> duplicates? -> "shop/kiosk|Роспечать"
  "shop/newsagent|Союзпечать" -> duplicates? -> "shop/kiosk|Союзпечать"
  "shop/perfumery|Marionnaud" -> duplicates? -> "shop/cosmetics|Marionnaud"
  "shop/supermarket|Alfamart" -> duplicates? -> "shop/convenience|Alfamart"
  "shop/supermarket|Bim" -> duplicates? -> "shop/supermarket|BIM"
  "shop/supermarket|Bodega Aurrera Express" -> duplicates? -> "shop/supermarket|Bodega Aurrera"
  "shop/supermarket|CBA" -> duplicates? -> "shop/convenience|CBA"
  "shop/supermarket|CU" -> duplicates? -> "shop/convenience|CU"
  "shop/supermarket|Casino" -> duplicates? -> "shop/convenience|Casino Shop"
  "shop/supermarket|Centra" -> duplicates? -> "shop/convenience|Centra"
  "shop/supermarket|Coles" -> duplicates? -> "shop/convenience|Coles Express"
  "shop/supermarket|Costcutter" -> duplicates? -> "shop/convenience|Costcutter"
  "shop/supermarket|Covirán" -> duplicates? -> "shop/supermarket|Coviran"
  "shop/supermarket|Crai" -> duplicates? -> "shop/supermarket|CRAI"
  "shop/supermarket|Delikatesy Centrum" -> duplicates? -> "shop/convenience|Delikatesy Centrum"
  "shop/supermarket|Franprix" -> duplicates? -> "shop/convenience|Franprix"
  "shop/supermarket|Fresh Market" -> duplicates? -> "shop/convenience|Freshmarket"
  "shop/supermarket|GS25" -> duplicates? -> "shop/convenience|GS25"
  "shop/supermarket|Groszek" -> duplicates? -> "shop/convenience|Groszek"
  "shop/supermarket|Hruška" -> duplicates? -> "shop/convenience|Hruška"
  "shop/supermarket|Indomaret" -> duplicates? -> "shop/convenience|Indomaret"
  "shop/supermarket|Joker" -> duplicates? -> "shop/convenience|Joker"
  "shop/supermarket|Konsum" -> duplicates? -> "shop/supermarket|Coop Konsum"
  "shop/supermarket|Konzum" -> duplicates? -> "shop/convenience|Konzum"
  "shop/supermarket|Londis" -> duplicates? -> "shop/convenience|Londis"
  "shop/supermarket|M&S Simply Food" -> duplicates? -> "shop/convenience|M&S Simply Food"
  "shop/supermarket|Mercator" -> duplicates? -> "shop/convenience|Mercator"
  "shop/supermarket|Petit Casino" -> duplicates? -> "shop/convenience|Petit Casino"
  "shop/supermarket|Prix" -> duplicates? -> "shop/supermarket|Coop Prix"
  "shop/supermarket|Proxi" -> duplicates? -> "shop/convenience|Proxi"
  "shop/supermarket|Studenac" -> duplicates? -> "shop/convenience|Studenac"
  "shop/supermarket|Vival" -> duplicates? -> "shop/convenience|Vival"
  "shop/supermarket|Volg" -> duplicates? -> "shop/convenience|Volg"
  "shop/supermarket|АТБ" -> duplicates? -> "shop/convenience|АТБ"
  "shop/supermarket|Авоська" -> duplicates? -> "shop/convenience|Авоська"
  "shop/supermarket|Верный" -> duplicates? -> "shop/convenience|Верный"
  "shop/supermarket|Гастроном" -> duplicates? -> "shop/convenience|Гастроном"
  "shop/supermarket|Гроздь" -> duplicates? -> "shop/convenience|Гроздь"
  "shop/supermarket|Евроопт" -> duplicates? -> "shop/convenience|Евроопт"
  "shop/supermarket|Квартал" -> duplicates? -> "shop/convenience|Квартал"
  "shop/supermarket|Кировский" -> duplicates? -> "shop/convenience|Кировский"
  "shop/supermarket|Магазин" -> duplicates? -> "shop/convenience|Магазин"
  "shop/supermarket|Мария-Ра" -> duplicates? -> "shop/convenience|Мария-Ра"
  "shop/supermarket|Монетка" -> duplicates? -> "shop/convenience|Монетка"
  "shop/supermarket|Семейный" -> duplicates? -> "shop/convenience|Семейный"
  "shop/supermarket|хүнсний дэлгүүр" -> duplicates? -> "shop/supermarket|Хүнсний дэлгүүр"
  "shop/tobacco|Nemzeti Dohánybolt" -> duplicates? -> "shop/kiosk|Nemzeti Dohánybolt"
total 111
Index completeness: 2548/3389 (75.2%) matched to Wikidata 

merging config/canonical.json
```
And I don't sure. What should I do there? Also feel free to add your commits because the option "Allow edits from maintainers."  is enabled on Github.

**Comment by Adamant36** [comment]
The list of tags is normal and can mostly be ignored. The merge conflict I'm not to sure about the merge conflict though.

File: config/canonical.json (hunk 2 above)
Comment (sguinetti) [reply config/canonical.json]: I installed nodejs and npm, and added the build-npm-apm extension in atom IDE. I cloned from Github and I ran the "build" function. That IDE eliminates the final space while. And this code is built successfully. Still, I do not know the next step.

**Comment by bhousel** [comment]
Seems fine now, I don't see a merge conflict anymore.. Thanks @sguinetti  👍

**Pull request merged**